doc/man3/ASN1_STRING_length.pod

```diff
@@ -71,8 +71,8 @@ utility functions should be used instead.
 In general it cannot be assumed that the data returned by ASN1_STRING_data()
 is null terminated or does not contain embedded nulls. The actual format
 of the data will depend on the actual string type itself: for example
-for and IA5String the data will be ASCII, for a BMPString two bytes per
-character in big endian format, UTF8String will be in UTF8 format.
+for IA5String the data will be ASCII, for BMPString two bytes per
+character in big endian format and UTF8String will be in UTF8 format.
```

Comment:
IMHO this patch removes one typo and adds two grammatical errors instead. Shouldn't it be more like this?
```
+for an IA5String the data will be ASCII, for a BMPString two bytes per
+character in big endian format, and for an UTF8String it will be in UTF8 format.
```
where the last "it" could also be replaced by "the data".

Reply:
I reported #5454 and I agree. I think the only change that needs to be done here is s/and/a/ or s/and/an/.

Reply:
Well, I think that

>  ... and UTF8String will be in UTF8 format.

is not gramatically correct and should also be corrected in the course of this pull request. For example, to

>  ... and for an UTF8String it will be in UTF8 format.

where 'it' refers to 'data', see the rest of the sentence.